tools/actions/commit-queue.sh

```diff
@@ -87,22 +87,18 @@ for pr in "$@"; do
     commit_body=$(git log -1 --pretty='format:%b')
     commit_head=$(grep 'Fetched commits as' output | cut -d. -f3 | xargs git rev-parse)
 
-    jq -n \
-      --arg title "${commit_title}" \
-      --arg body "${commit_body}" \
-      --arg head "${commit_head}" \
-      '{merge_method:"squash",commit_title:$title,commit_message:$body,sha:$head}' > output.json
-    cat output.json
-    if ! gh api -X PUT "repos/${OWNER}/${REPOSITORY}/pulls/${pr}/merge" --input output.json > output; then
+    if ! commits="$(
+      jq -n \
```

Comment:
Nit: if the request body is not meant to be printed anymore (otherwise we can use `tee` to avoid temporary file), there's no reason to pretty-print it.
```suggestion
      jq -cn \
```